src/Pages/Channels.razor

```diff
@@ -566,14 +566,15 @@
         arg1.Status = ValidationStatus.Success;
 
     //If the minimum local balance is 0 this cannot be 0
-        if (_currentLiquidityRule.MinimumLocalBalance == 0 && _currentLiquidityRule.MinimumRemoteBalance == 0)
+        if ((_currentLiquidityRule.MinimumLocalBalance == 0 || _currentLiquidityRule.MinimumLocalBalance == null) 
+            && (_currentLiquidityRule.MinimumRemoteBalance == 0 || _currentLiquidityRule.MinimumRemoteBalance == null))
         {
             arg1.Status = ValidationStatus.Error;
             arg1.ErrorText = "Minimum remote balance cannot be 0 if the minimum local balance is 0";
         }
 
     //If the value is 0 is valid
-        if (_currentLiquidityRule.MinimumRemoteBalance == 0)
+        if (_currentLiquidityRule.MinimumRemoteBalance == 0 || _currentLiquidityRule.MinimumRemoteBalance == null)
```

Comment:
BTW @Jossec101 
Here you are not reaching to check that the Remote balance is 0 > x < 100. Is this okay or is it a bug?